templates/form_worldcat.html

```diff
@@ -1,13 +1,12 @@
-<p>Search books and other media worldwide.</p>
 <form action="https://mit.worldcat.org/search" method="get" id="booksearch" data-target="worldcat">
 	<div class="hidden">
 		<input type="hidden" name="qt" value="wc_org_mit">
 		<input type="hidden" name="qt" value="affiliate">
 		<input type="hidden" name="q">
 	</div>
 	<div class="row">
-		<input type="text" class="request" name="search" placeholder="Search books, films, music scores, maps...">
-		<input type="submit">
+		<input type="text" class="request" name="search" placeholder="Search books, films, music scores...">
+		<input type="submit" value="Search">
```

Comment:
Same here: I would rather have the submit semantically last and work on visual arrangement with css. I think you can drop the "row" div and move submit after the type selector.